macaroon.js

```diff
@@ -505,7 +505,7 @@ const decrypt = function(keyBits, ciphertextBits) {
   const nonceBytes = ciphertextBytes.slice(0, NONCELEN);
   const dataBytes = ciphertextBytes.slice(NONCELEN);
   let textBytes = nacl.secretbox.open(dataBytes, nonceBytes, keyBytes);
-  if (textBytes === false) {
+  if (textBytes === false || textBytes === null) {
```

Comment:
Is false actually a possibility here? [This documentation](https://github.com/dchest/tweetnacl-js#secret-key-authenticated-encryption-secretbox) seems to indicate that it always returns null on failure.

Assuming it can return either value, would it be more JS-idiomatic to do:

    if(!testBytes) {

?
@hatched

Reply:
That would also match on an empty string so I feel like being explicit on the type here would be useful, even if just for others to grok the code.

It looks like it used to return false, and now with v1 returns null https://github.com/dchest/tweetnacl-js/commit/8626b70ad7a242fe4516e3bcba212b955968c967

Because we have the version locked to above 1 removing that false check is probably safe to do.

Reply:
I believe I've made the requested change, let me know if you need anything else from me. Thanks!